### ISSUE TITLE
Some text is not translated

### ISSUE THREAD

**Issue by mikhailov-sergey** [qgis/QGIS#37558]
**Describe the bug**

We are translating the QGIS interface into Russian. There was a problem that menu items that have the horizontal ellipsis symbol '…' (\u2026) are not translated into either Russian or other languages. But if you replace this character with a simple ellipsis '...', making a corresponding change in the translation file, then the problem disappears.

**How to Reproduce**
1. Switch language (to other than English) and restart QGIS.
2. For example: dlick on the fifth tab on the menu bar('Settings' in English)
3. See error --> items 'Style Manager…', 'Custom Projections…' and other are not translated.

**QGIS and OS versions**

QGIS version: 3.15.0-Master
OS: Ubuntu 18.04

**Additional context**

As one of the possible solutions, we can consider replacing the horizontal ellipsis symbol '…' (\u2026) with a simple ellipsis '...' in the entire interface. We are ready to do this if in the future you will accept these changes to the main project code.

**Comment by roya0045** [comment]
Yeah those were changed to the unicode one for translating/display reasons also if I recall. 
See the following
https://github.com/qgis/QGIS/pull/31124
https://github.com/qgis/QGIS/pull/33331
https://github.com/qgis/QGIS/pull/31895

**Comment by mikhailov-sergey** [comment]
We did not try to translate this symbol, we added only missing translations to the qgis_ru.ts file. Some translations are missing, although a translation has been added to the file.
Example:
OS: KDE neon User Edition 5.19
Qt: 5.14.2
QGIS: 3.15.0-Master

![screen](https://user-images.githubusercontent.com/58285642/86577014-43100b00-bf82-11ea-9da1-44579abaf556.png)

**Comment by Pedro-Murteira** [comment]
This seems to be the case with some languanges, but not all of them. For example, I've tested this with Portuguese, French, Spanish, Italian and Deutsch and the translations are correct. On the other hand, there are some languages with the issue reported here, for example: Greek, Indonesian and more.

**Comment by Geoscope19** [comment]
Hello,
A typo in the French translation: "chaNger" instead of "chaRger"
QGis 3.30.1 fr pour windows
![Sans titre](https://user-images.githubusercontent.com/81222128/234002360-c68688a4-f9ae-426d-b121-556d88f66482.png)

**Comment by nicogodet** [comment]
@Geoscope19 No typo here "Changer..." is correct -> "Change CRS"

C'est bien "Changer" et non "Charger" en référence à "Changer le système de coordonnées de référence".

**Comment by GevaGis** [comment]
hi, when you click on the screen button check in the filter, instead of the number of returned rows, the message “% of rows returned” appears
